### PR TITLE
Add IPVanish and NordVPN

### PR DESCRIPTION
**Pull request by spazvt**
add IPVanish and NordVPN service provider, originally added by @deadr0ck to fork
https://github.com/acceptbitcoincash/acceptbitcoincash

### CODE REVIEW

File: _data/security.yml [hunk 1]
```diff
@@ -90,6 +90,24 @@ websites:
...
+      doc: https://blog.ipvanish.com/lock-down-logins-5-password-tips/
+
+    - name: NordVPN
+      url: https://www.nordvpn.com
+      img: nordvpn.png
+      tfa: Yes
+      software: Yes
+      twitter: nordvpn
+      facebook: NordVPN
+      doc: https://nordvpn.com/blog/secure-accounts-two-factor-authentication/
```
Comment (SonnevilleJ) [inline _data/security.yml]: This linked doc does not include instructions to enable two-factor authentication for NordVPN.

**Review by SonnevilleJ — CHANGES_REQUESTED** [review]

File: _data/security.yml (hunk 1 above)
Comment (Carlgo11) [reply _data/security.yml]: I can confirm that NordVPN doesn't offer two factor authentication to it's users.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by Carlgo11** [comment]
Closing due to inactivity

**Comment by spazvt** [comment]
@Carlgo11 and @SonnevilleJ you're right, it looks like that NordVPN page only gives instructions for other websites and not its own. Since @Carlgo11 says it definitely doesn't offer 2FA, I can change the entry to list that.

**Comment by spazvt** [comment]
PR updated. please re-review.

**Comment by spazvt** [comment]
any updates on reviewing this PR, @Carlgo11 @mxxcon @jamcat22

**Review by mxxcon — CHANGES_REQUESTED** [review]
Please use Nord logo from their website https://nordvpn.com/wp-content/themes/nordvpn-bootstrap/build/1524660327/images/favicons/favicon-32x32.png

**Comment by spazvt** [comment]
@mxxcon requested update made.

**Review by mxxcon — DISMISSED** [review]

File: _data/security.yml [hunk 2]
```diff
@@ -105,12 +105,26 @@ websites:
       img: hidemevpn.png
       tfa: No
 
+    - name: IPVanish
+      url: https://www.ipvanish.com
+      img: ipvanish.png
+      tfa: No
+      twitter: ipvanish
```
Comment (Carlgo11) [inline _data/security.yml]: Please change the Twitter tag to `IPVanish`.

File: _data/security.yml [hunk 3]
```diff
@@ -105,12 +105,26 @@ websites:
...
       url: https://www.ivpn.net/
       twitter: ivpnnet
       img: ivpn.png
       tfa: No
 
+    - name: NordVPN
+      url: https://www.nordvpn.com
+      img: nordvpn.png
+      tfa: No
+      twitter: nordvpn
```
Comment (Carlgo11) [inline _data/security.yml]: Please change the Twitter handle to `NordVPN`.

File: _data/security.yml [hunk 4]
```diff
@@ -105,12 +105,26 @@ websites:
...
+      facebook: ipvanish
+
     - name: IVPN
       url: https://www.ivpn.net/
       twitter: ivpnnet
       img: ivpn.png
       tfa: No
 
+    - name: NordVPN
+      url: https://www.nordvpn.com
```
Comment (Carlgo11) [inline _data/security.yml]: Change the url to `https://nordvpn.com/`

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by spazvt** [comment]
@Carlgo11 all requested changes made. Anything else?

**Review by Carlgo11 — APPROVED** [review]
Looks good

**Review by jamcat22 — APPROVED** [review]
Thanks for the PR @spazvt!

**Pull request merged**